### ISSUE TITLE
InvalidArgumentError when fine-tuning inception_v3 on flowers

### ISSUE THREAD

**Issue by trungdn** [tensorflow/models#619]
I'm running TensorFlow 0.11.0rc0  in ubuntu 14.04.

When I tried to fine-tuning Inception_v3 on flowers dataset follow that command:
python train_image_classifier.py \
    --train_dir=${TRAIN_DIR} \
    --dataset_dir=${DATASET_DIR} \
    --dataset_name=flowers \
    --dataset_split_name=train \
    --model_name=inception_v3 \
    --checkpoint_path=${CHECKPOINT_PATH} \
    --checkpoint_exclude_scopes=InceptionV3/Logits,InceptionV3/AuxLogits/Logits \
    --trainable_scopes=InceptionV3/Logits,InceptionV3/AuxLogits/Logits

It shows the errors:
InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [1,1,768,5] rhs shape= [1,1,768,1001]

I thought the flags ----checkpoint_exclude_scopes will avoid that unbalance shapes.
Thanks

**Comment by plediii** [comment]
There was a typo on the scopes in the README.  See https://github.com/tensorflow/models/blob/master/slim/scripts/finetune_inception_v3_on_flowers.sh#L45

**Comment by ahundt** [comment]
This is still outstanding and I've tested that the pull request #632 works. @girving or @sguada is it possible to merge #632 and close this issue?

**Comment by ozancaglayan** [comment]
Same typo exists in https://github.com/tensorflow/models/blob/master/slim/README.md as well.
